source/wp-content/themes/wporg-news-2021/block-template-parts/footer-archive.html

```diff
@@ -0,0 +1,75 @@
+<!-- wp:group {"align":"full","className":"footer-archive-row","layout":{"type":"flex"}} -->
+<div class="wp-block-group alignfull footer-archive-row">
+	<!-- wp:group -->
+	<div class="wp-block-group">
+		<!-- wp:heading {"level":4} -->
+		<h4>Follow The Code</h4>
+		<!-- /wp:heading -->
+		<!-- wp:paragraph -->
+		<p><a href="https://make.wordpress.org/core/">There's a development P2</a> blog and you can track active development in the <a href="https://core.trac.wordpress.org/timeline">Trac timeline</a> that often has 20–30 updates per day.</p>
+		<!-- /wp:paragraph -->
+	</div>
+	<!-- /wp:group -->
+	<!-- wp:group -->
+	<div class="wp-block-group">
+		<!-- wp:heading {"level":4} -->
+		<h4>Find An Event Near You</h4>
+		<!-- /wp:heading -->
+		<!-- wp:paragraph -->
+		<p>Check out the <a href="https://central.wordcamp.org/">WordCamp schedule</a> and find your <a href="https://www.meetup.com/pro/wordpress">local Meetup group</a>! For more WordPress news, check out the <a href="http://planet.wordpress.org/">WordPress Planet</a>.</p>
+		<!-- /wp:paragraph -->
```

Comment:
In the future we could maybe use the Events Widget to show actual events here instead, but that'd definitely be a separate pr